nixos/modules/security/ipa.nix

```diff
@@ -279,13 +282,13 @@ in
 
         ifp = {
           user_attributes = "+mail, +telephoneNumber, +givenname, +sn, +lock";
-          allowed_uids = concatStringsSep ", " cfg.ifpAllowedUids;
+          allowed_uids = lib.concatStringsSep ", " cfg.ifpAllowedUids;
         };
       };
     };
 
-    networking.timeServers = singleton cfg.server;
+    networking.timeServers = lib.singleton cfg.server;
```

Comment:
from beginning setting ntp make no sense to me

Reply:
see #461739